api/desecapi/migrations/0013_acme_challenge.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.3 on 2016-12-27 07:59
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('desecapi', '0012_move_dyn_flag'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='domain',
+            name='acme_challenge',
+            field=models.CharField(max_length=255, blank=True),
```

Comment:
length set to 255, however it is unclear if bytes or characters are counted (UTF-8)